mne/io/artemis123/utils.py

```diff
@@ -82,3 +84,55 @@ def _compute_mne_loc(coil_loc):
     R = rotation3d_align_z_axis(z_axis)
     loc[3:13] = R.T.reshape(9)
     return loc
+
+
+def _read_pos(fname):
```

Comment:
@bloyl I think that so far the Digitization refactoring plans would be mostly independent of these CTF issues. Can you open a new issue for these potential problems? We'll probably need to discuss them a bit (perhaps with Matti, too) and sort them out. Then we can unify here and in the CTF code.